### ISSUE TITLE
Support for openmaptiles

### ISSUE THREAD

**Issue by nyurik** [maplibre/martin#20]
The [openmaptiles](https://github.com/openmaptiles/openmaptiles) project also supports postgis-based tile generation - it can generate giant SQL statement (as either prepared statement, query, or function creation) that produces MVT from z,x,y.   Would it be possible to add support for the openmaptiles layer format in martin as well?  See [the postgis server](https://github.com/openmaptiles/openmaptiles-tools#test-tiles) (simple test server for testing purposes)

**Comment by stepankuzmin** [comment]
Hi @nyurik! That's a great idea! Unfortunately, I don't have enough bandwidth for implementing this now. But I'll keep it in mind and maybe implement this in the future releases.

**Comment by littlebtc** [comment]
After trying OpenMapTiles, I found that the docker-compose file from https://github.com/openmaptiles/openmaptiles/ still using old PostGIS with old `ST_AsMVT` syntax. A major PostGIS update for OpenMapTiles is needed in order to run it :(

**Comment by nyurik** [comment]
@littlebtc most of the tools have already been updated to the latest. https://github.com/openmaptiles/openmaptiles-tools has been receiving tons of changes in the last weeks (just today I made a number of changes to postserve -- a simple debug-only version of `martin` that uses OpenMapTile layer defenition)

I have been doing some of it as part of https://github.com/sophox/openmaptiles too, and very soon I plan to have a working docker image with PostgreSQL 12 + PostGIS 3.0 (they are massively faster for MVTs, and were released a week ago).   My suggestion - for testing, simply swap the order of the parameters to ST_AsMVT -- and your code will work right away with the existing quick start. I will have the quickstart ready within the next few days to run on the latest platform.

Thanks!
